webapp/src/components/help.md

```diff
@@ -0,0 +1,119 @@
+# Welcome to the {EnvironmentVars.DOT_NAME} CV Manager Website
+
+This application helps organizations manage and monitor deployed RSUs, monitor and detect issues with connected intersections, and view numerous data types all in one application.
+
+## Organizations and Profiles
+
+This application uses organizations to manage permissions. All devices and users within the CV Manager are associated with 1 or more organizations. When using the CV Manager, you can only view one organization at a time. At the top right corner, you will see the User Profile menu. This menu opens to allow the user to change organizations, if they are a member or multiple, as well as logging out of the application.
+
+<img src=../icons/help/organization_selection.png alt="Profile Dropdown" width="200"/>
+
+**Note on feature flags**
+
+Some features in the CV Manager can be enabled or disabled for each version. The feature flags set in this application are:
+
+- ENABLE_RSU_FEATURES: ENABLED
+  - Affects: RSU viewer, RSU status monitoring, RSU configuration
+- ENABLE_INTERSECTION_FEATURES: ENABLED
+  - Affects: Intersection map, Intersection dashboard, Conflict Monitor events, assessments, and notifications
+- ENABLE_WZDX_FEATURES: DISABLED
+  - Affects: WZDx message viewer
+- ENABLE_MOOVE_AI_FEATURES: DISABLED
+  - Affects: Moove AI hard braking event viewer
+
+## [Map Dashboard](/dashboard/map)
+
+The map dashboard is composed of a Mapbox map (background), the [Map Layer Menu](#map-layers) (red), and [RSU Status and Message Counts Menu](#rsu-status-and-message-counts) (green)
+
+<img src=../icons/help/map_overview.png alt="Map Layer Selection Options" width="1000"/>
+
+### Map Layers
+
+The menu on the left side of the screen contains three main sections: Map Layers, RSU Filters, and RSU
+Configuration. The Map Layers section allows users to change how data is visualized. The available layers are:
+
+- RSU Viewer: display location and status of RSUs. Selecting an RSU will open a popup with status and health information, as well as opening a side panel for RSU configuration. The color of the RSU points indicates the status (configurable under "Filter RSUs"). Green means the RSU is online and actively reporting data, yellow indicates the RSU is currently offline but was recently active, and red indicates that RSU is offline and has not reported in some time.
+- Heatmap: shows a heatmap based on message counts using the filters from the Message Count Menu
```

Comment:
Can period usage in this bullet list be made consistent? The first one has them since it is multiple sentences but these are complete sentences and could be consistent with them.